Loop/Managers/Store Protocols/DosingDecisionStoreProtocol.swift

```diff
@@ -6,16 +6,28 @@
 //  Copyright © 2020 LoopKit Authors. All rights reserved.
 //
 
+import LoopAlgorithm
 import LoopKit
 
+struct LightDosingDecision: DosingDecision {
```

Comment:
Is this needed, or can the other code just access the objects returned from the store as `DosingDecision` types?

Reply:
`DosingDecision ` is a protocol I introduced into LoopKit but I can move this there instead!

Reply:
That's fine; was just checking.